doc/cli.markdown

```diff
@@ -2509,7 +2509,7 @@ any) defined at the overall project root will be used to filter files and
 subdirectories other than service subdirectories. It will not have any effect
 on service subdirectories, whether or not a service subdirectory defines its
 own .dockerignore file. Multiple .dockerignore files are not merged or added
-together, and cannot override or extend other files. This behavior maximises
+together, and cannot override or extend other files. This behavior maximizes
```

Comment:
`cli.markdown` is auto generated though. You need to search and find where this text comes from in source code (Typescript or Javascript), then run `npm run build` to refresh `cli.markdown` and commit.

Reply:
good catch, thanks!